standalone/standalone.py

```diff
@@ -97,6 +97,9 @@
 MERGE_SYSTEM_USER_MESSAGE = False
 FEW_SHOTS = 5
 BATCH_SIZE = 8
+JUDGE_CA_CERT_ENV_VAR_NAME = "JUDGE_CA_CERT_PATH"
+JUDGE_CA_CERT_PATH = "/tmp/judge_ca_cert.pem"
+JUDGE_CA_CERT_SECRET_KEY = "ca-bundle.crt"
```

Comment:
you can actually just utilize the name of the key as the file name, by doing the following: 


```yaml
volumes:
        - name: "judge-ca-cert-volume"
          configMap:
            name: ${judge-serving-model-ca-cert}
```
and then in your volume mount: 

```yaml
        volumeMounts:
          - mountPath: /some/path/           # no file name
            name: "judge-ca-cert-volume"
```

Then the full path would be: ${JUDGE_CA_CERT_PATH}/${JUDGE_CA_CERT_SECRET_KEY}

Reply:
Thanks, I removed `sub_path` and used `/tmp`.